src/UI/Component/Listing/Factory.php

```diff
@@ -46,4 +46,28 @@ public function ordered(array $items);
 	 */
 	public function descriptive(array $items);
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      A workflow presents a list of steps that the user should tackle
+	 *      in a defined order.
+	 *   composition: >
+	 *     The workflow has a title and a list of workflow steps.
+	 *   effect: >
+	 *     Steps in a workflow reflect their progress (not applicable, not started, in progress, completed).
```

Comment:
Nice!

Reply:
How will these Progress status be determined? Why is this sophistication needed.